src/main.rs

```diff
@@ -18,6 +24,16 @@ fn main() -> std::io::Result<()> {
     std::env::set_var("RUST_LOG", &settings.log_level);
     env_logger::init();
 
+    #[cfg(feature = "consensus")]
+    {
+        // `raft` crate uses `slog` crate so it is needed to use `slog_stdlog::StdLog` to forward
+        // logs from it to `log` crate
+        let slog_logger = slog::Logger::root(slog_stdlog::StdLog.fuse(), slog::o!());
+
+        let (mut consensus, _message_sender) = Consensus::new(&slog_logger);
+        thread::spawn(move || consensus.start());
```

Comment:
Would be great to give a name to this thread just in this case it starts acting weird.